src/materials/DamagePlasticityStressUpdate.C

```diff
@@ -318,16 +309,14 @@ DamagePlasticityStressUpdate::flowPotential(const std::vector<Real> & stress_par
                  RankTwoTensor(stress_params[0], stress_params[1], stress_params[2], 0, 0, 0)
                      .dsecondInvariant();
 
-  Real D = damageVar(stress_params, intnl);
-
   for (unsigned int i = 0; i < _num_sp; ++i)
-    r[i] = (_alfa_p + d_sqrt_2J2(i, i)) * (1. - D);
+    r[i] = (_alfa_p + d_sqrt_2J2(i, i));
 }
 
 void
 DamagePlasticityStressUpdate::dflowPotential_dstress(
     const std::vector<Real> & stress_params,
-    const std::vector<Real> & intnl,
+    const std::vector<Real> & /* intnl */,
```

Comment:
Should we just remove this from the parameter list entirely if it's not used?

Reply:
Addressed